docs/hazmat/primitives/asymmetric/ec.rst

```diff
@@ -704,6 +704,27 @@ Key Interfaces
         Size (in :term:`bits`) of a secret scalar for the curve (as generated
         by :func:`generate_private_key`).
 
+    .. classmethod:: from_encoded_point(curve, data, backend)
```

Comment:
Why does this take a backend? Aren't we over this?

Reply:
I don't think I'm comfortable removing backend from a single classmethod without deprecating it on all EC things? I'm open to being convinced otherwise but it makes me uncomfortable to piecemeal it for some reason.

Reply:
We've already been not adding it on new APIs, this is a new API.